src/unix/fsevents.c

```diff
@@ -267,12 +271,6 @@ static void uv__fsevents_event_cb(ConstFSEventStreamRef streamRef,
         }
       }
 
-#ifdef MAC_OS_X_VERSION_10_7
-      /* Ignore events with path equal to directory itself */
-      if (len == 0)
```

Comment:
A few of the tests fail with this. The file changed events are never received with this still in. Is there a specific case where an extra event for directories would show up? or is it in all cases?